### ISSUE TITLE
vcf output has either very high tumor and low normal coverage or the other way around

### ISSUE THREAD

**Issue by wtwt5237** [Illumina/strelka#20]
Hello!

I am using strelka2 for calling indels. The indels called by other software and strelka1 all look normal. However, when I look into the output of strelka2, all variants have either very high tumor and very low normal coverage, or the other way around. And they have almost no overlap with the indels called by other software. After some obvious filtering, zero valid mutations called by strelka2 will be left.

Please see attached file. It looks like there is a bug in strelka2.

[somatic.indels.txt](https://github.com/Illumina/strelka/files/1423306/somatic.indels.txt)

Thanks!

Tao

**Comment by sangtaekim** [comment]
For non-WGS data, it is required to provide the "--exome" flag at configuration. Otherwise, valid variants will be filtered out by Strelka's high-depth filters that expects a super low depth at every locus. I believe re-running Strelka2 with the "--exome" flag will bring in valid mutations.

Still, the FPs in your results look problematic and we will fix it. If you can send us your BAM files, that will be very helpful.

**Comment by wtwt5237** [comment]
Hi Sangtae

Thanks a lot for your reply!

I am using RNA-Seq data for somatic mutation calling. So I should still turn on –exome? Any other parameters to turn on/off?

How can I send you the bam files?

Tao

**Comment by sangtaekim** [comment]
Yes, you have to set "--exome" for RNA-Seq data and that will be sufficient. Note that Strelka2 doesn't "officially" support RNA-Seq data for somatic calling, so the results may not be optimal.

For uploading the files, I'll email you a Box link.
